src/neynar-api/v2/openapi-farcaster/apis/user-api.ts

```diff
@@ -1230,6 +1318,22 @@ export class UserApi extends BaseAPI {
     public userBulkByAddress(addresses: string, addressTypes?: string, viewerFid?: number, options?: RawAxiosRequestConfig) {
         return UserApiFp(this.configuration).userBulkByAddress(addresses, addressTypes, viewerFid, options).then((request) => request(this.axios, this.basePath));
     }
+    
+    /**
+    * Fetches a list of users given a location
+    * @summary By location
+    * @param {number} latitude Latitude of the location
+    * @param {number} longitude Longitude of the location
+    * @param {number} [viewerFid] FID of the user viewing the feed. Providing this will return a list of users that respects this user\&#39;s mutes and blocks and includes &#x60;viewer_context&#x60;.
+    * @param {number} [limit] Number of results to fetch
```

Comment:
Please mention the default and max limit in the OAS as well. it'll be helpful for v2 code gen